ReactWindows/ReactNative.Shared/Views/Text/FontStyleHelpers.cs

```diff
@@ -8,6 +9,23 @@ namespace ReactNative.Views.Text
 {
     static class FontStyleHelpers
     {
+        public static FontStyle? ParseFontStyle(string fontStyleString)
+        {
+#if WINDOWS_UWP
+            return EnumHelpers.ParseNullable<FontStyle>(fontStyleString);
+#else
+            if (fontStyleString == "normal")
+            {
+                return FontStyles.Normal;
+            }
+            else if (fontStyleString == "italic")
+            {
+                return FontStyles.Italic;
+            }
+            return null;
```

Comment:
bold is in fontWeight, please see https://facebook.github.io/react-native/docs/text.html#style